doxygen/examples/CMakeTests.cmake

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright by The HDF Group.
+# All rights reserved.
+#
+# This file is part of HDF5.  The full HDF5 copyright notice, including
+# terms governing use, modification, and redistribution, is contained in
+# the COPYING file, which can be found at the root of the source code
+# distribution tree, or in https://www.hdfgroup.org/licenses.
+# If you do not have access to either file, you may request a copy from
+# help@hdfgroup.org.
+#
+
+##############################################################################
+##############################################################################
+###           T E S T I N G                                                ###
+##############################################################################
+##############################################################################
+file (MAKE_DIRECTORY ${PROJECT_BINARY_DIR}/red ${PROJECT_BINARY_DIR}/blue ${PROJECT_BINARY_DIR}/u2w)
```

Comment:
Does the example expect these dirs?

Reply:
Honestly, I believe I copied this file over from `./examples/CMakeTests.cmake`, so this may unnecessary here.

Reply:
Fixed in [12fbb84](https://github.com/HDFGroup/hdf5/pull/2380/commits/12fbb846aae95b34806dadf56507c87b647bf373)